dnscache/storage.go

```diff
@@ -0,0 +1,94 @@
+package dnscache
+
+import (
+	"net"
+	"reflect"
+	"time"
+
+	"fmt"
+
+	"github.com/pmylund/go-cache"
+)
+
+// DnsCacheItem represents single record in cache
+type DnsCacheItem struct {
+	addrs []string
+}
+
+func (item *DnsCacheItem) IsEqualsTo(addrs []string) bool {
```

Comment:
don't add to main code things like this - code pieces needed to run unit-tests

Reply:
Agree. Main code should not contain helpers used for tests. Needs be moved to test suite.